api/core/v1alpha1/catalog_types.go

```diff
@@ -134,6 +134,10 @@ type ImageSource struct {
 	// specified as a duration (e.g., "5m", "1h", "24h", "etc".). Note that PollInterval may not be
 	// specified for a catalog image referenced by a sha256 digest.
 	PollInterval *metav1.Duration `json:"pollInterval,omitempty"`
+	// Insecure indicates that the registry is an insecure registry and that TLS verification
+	// should be skipped
+	// +optional
+	Insecure bool `json:"insecure,omitempty"`
```

Comment:
Make this field name a bit more descriptive?
```suggestion
	TLSInsecureSkipVerify bool `json:"tlsInsecureSkipVerify,omitempty"`
```

Reply:
Done

Reply:
Renamed again to `SkipTLSVerify` as this setting doesn't enforce using HTTP and will still result in the use of an HTTP client. If we want to add a method for forcing use of HTTP we should investigate using a configuration option for specifying a global set of insecure registries

Reply:
This is still insecure though because it means that an attacker could man-in-the-middle the connection and use their own certificate to see and manipulate the request/response.

I think `Insecure` needs to be in the name.

Reply:
`InsecureSkipTLSVerify` please - that aligns with other names: https://github.com/search?q=insecureskiptlsverify&type=code

Reply:
`InsecureSkipTLSVerify` in the latest push